src/views/search/SearchResultsView.integrationTest.tsx

```diff
@@ -297,6 +297,13 @@ describe('SearchResults', () => {
       });
 
       expect(wrapper.getByText('Remove')).toBeInTheDocument();
+      const addNotification = wrapper.getByTestId(
```

Comment:
Instead of using getByTestId, could we use `getByRole('alert')` or maybe even do `getByRole('alert', {name: "Video added to cart"})`